docs/content/operations/api-reference.md

```diff
@@ -230,6 +230,20 @@ Enables all segments of datasource which are not overshadowed by others.
 
 Enables a segment of a datasource.
 
+##### PUT
```

Comment:
I was thinking the drop is not really idempotent, since it can have different effects each call (suppose new segments were added to the drop interval between calls)